test/quantization/dbr/test_quantize_dbr.py

```diff
@@ -1023,16 +1023,16 @@ def forward(self, x):
 
         m = M().eval()
         qconfig = torch.quantization.default_qconfig
-        example_args = (torch.randn(1, 1, 2, 2),)
-        mp = _quantize_dbr.prepare(m, {'': qconfig}, example_args)
-        out_p = mp(*example_args)
+        example_inputs = (torch.randn(1, 1, 2, 2),)
+        mp = _quantize_dbr.prepare(m, {'': qconfig}, example_inputs)
```

Comment:
seems like L85 is related to this PR, but this line and most/all lines below are not?  Maybe we can just stick with the change in L85 to ensure that PR is not changing unrelated things?

I think changing these var names to `example_inputs` also makes sense, it's just not intuitive for it to be in this PR.  Thoughts?

Reply:
sure, I can revert this for now